docs/4.2/admin-guide.md

```diff
@@ -1956,6 +1956,16 @@ To allow access for all users:
   + Copy `teleport-user-ca.pub` to `/etc/ssh/teleport-user-ca.pub`
   + Update `sshd` configuration (usually `/etc/ssh/sshd_config` ) to point to
     this file: `TrustedUserCAKeys /etc/ssh/teleport-user-ca.pub`
+ 
+ To connect to the OpenSSH server via `tsh` add the `--port=<ssh port>` with the `tsh ssh` command:
```

Comment:
...via `tsh`, add `--port=<ssh port>` to the `tsh ssh` command:

(add comma, fix formatting)